docker/images/dev-cli/Dockerfile

```diff
@@ -1,6 +1,12 @@
 # Creates an image for skycoin development
 FROM golang:1.11-stretch
 
+# Image labels
+LABEL "org.label-schema.name"="skycoindev-cli develop" \
```

Comment:
name = `skycoindev-cli`

[`http://label-schema.org/rc1/#label-semantics` SHOULD be present](http://label-schema.org/rc1/#label-semantics) . Some other labels that might be nice to have

- `org.label-schema.build-date`
- `org.label-schema.usage` with link to Docker image README . In this case use git tag in URL if applicable else SHA1 hash .
- `org.label-schema.url` => skycoin.net
- `org.label-schema.vcs-ref`
- `org.label-schema.vendor` => Skycoin project
- `org.label-schema.version` => if built from tag then set it to tag name 
- `org.label-schema.schema-version` =>  = 1.0
- `docker.cmd` , documented in README
- `docker.cmd.help` for `skycoin/skycoin` image
- `docker.params` maybe for `skycoin/skycoin`

These comments are extensible to other images .